source/index.html.md.erb

```diff
@@ -1,21 +1,33 @@
 ---
-title: GDS technical guidance
+title: The GDS Way
 ---
 
 # <%= current_page.data.title %>
 
+# What is The GDS Way?
+
+Since GDS was formed a few years ago, we've grown to of an organisation of several hundred people with several well established teams which are meeting user needs by creating software.
+
+As we've grown the teams have all had to choose how they want to build their services. In some ways this has been good - every team has picked the most suitable tools for their project - but now we suspect that we're at the point where having some easy-to-use defaults for new teams will be helpful. Some of the things we're hoping to achieve are:
+
+* using similar tools across the organisation will make people more productive when they move between teams, meaning we might be able to do that faster and more often (assuming people want to)
+* doing procurement once centrally for the whole organisation instead of each team needing to spend time justifying obvious decisions will save a lot of time
+* new teams will be cheaper to spin up than ever before because starting to run applications will involve setting up accounts rather than building infrastructure
```

Comment:
i think a bigger part of this is that teams won't have to spend time making inconsequential decisions - "where will we host"? "what format will we log in?" "what names do we use for metrics?" etc